sjb/config/test_cases/test_branch_origin_extended_conformance_install_update_system_containers.yml

```diff
@@ -0,0 +1,249 @@
+        sudo atomic pull --storage ostree docker:openshift/origin:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/openvswitch:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/node:$( cat ./ORIGIN_TAG )
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e openshift_use_system_containers=true \
+                         -e openshift_docker_systemcontainer_image_registry_override=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7 \
+                         -e system_images_registry=docker \
+                         -e containerized=true      \
+                         -e deployment_type=origin  \
+                         -e openshift_image_tag="$( cat ./ORIGIN_TAG )"                      \
+                         -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"            \
+                         -e oreg_url='openshift/origin-${component}:'"$( cat ./ORIGIN_TAG )" \
+                         -e openshift_node_port_range='30000-32000'                          \
+                         -e 'osm_controller_args={"enable-hostpath-provisioner":["true"]}'   \
+                         /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
+    - type: "script"
+      title: "ensure built version of origin is installed"
+      timeout: 600
+      repository: "aos-cd-jobs"
+      script: |-
+        if [[ -n "${PULL_REFS:-}" ]]; then
+          export OPENSHIFT_ANSIBLE_TARGET_BRANCH="${PULL_REFS%%:*}"
+        fi
+        if [[ "${OPENSHIFT_ANSIBLE_TARGET_BRANCH:-}" == "release-1.5" ]]; then
+            echo "[INFO] System containers are not supported, skipping."
+            exit 0
+        fi
+
+        docker_image_tag="$( cat ./ORIGIN_TAG )"
+        docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
+    - type: "script"
+      title: "update origin to release"
+      repository: "aos-cd-jobs"
+      timeout: 1800
+      script: |-
+        if [[ "${OPENSHIFT_ANSIBLE_TARGET_BRANCH:-}" == "release-1.5" ]]; then
+            echo "[INFO] System containers are not supported, skipping."
+            exit 0
+        fi
+        ansible-playbook -vv --become               \
+                         --become-user root         \
+                         --connection local         \
+                         --inventory sjb/inventory/ \
+                         -e openshift_use_system_containers=true \
+                         -e openshift_docker_systemcontainer_image_registry_override=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7 \
```

Comment:
Same issues as above.

Reply:
Renamed

Reply:
I *think* you'll end up getting ``latest`` if you leave off the tag but I'm not 100% sure. @giuseppe WDYT?

Reply:
What is the expected tag for the 3.7 and 3.6? latest and 3.6?

Reply:
yes, when `openshift_docker_systemcontainer_image_registry_override` is used then we use it as it is, without adding any tag (defaulting to `latest`).

I see no v3.6 tag on `brew-pulp-docker01`, so I guess using latest in both cases is fine.